pkg/release/create.go

```diff
@@ -18,12 +22,23 @@ import (
 func CreateRelease(name, base, releases, provider string, components, apps []string, overwrite bool, creationCommand string, bumpall bool) error {
 	// Paths
 	baseVersion := *semver.MustParse(base) // already validated to be a valid semver string
-	providerDirectory := filepath.Join(releases, provider)
+	providerDirectory := ""
+	if provider == "aws" {
```

Comment:
I guess this means that creating vintage releases won't work anymore. But we can still revert to old binaries to do that, or handcraft them, so it's fine IMO.

Reply:
Yeah, my thinking was we don't need another vintage release. Using an old binary was also my idea, I make it explicit in the changelog to use an older release of devctl if we need a new vintage release